### ISSUE TITLE
Plugin is not working properly.

### ISSUE THREAD

**Issue by DogancanYr** [webcrawls/discord#5]
I usually have to turn it off and on.

**Comment by webcrawls** [comment]
Hi there! I know there are a few outstanding issues with the plugin - if you could provide any reproduction steps, or give a bit of detail about the scenarios in which the plugin fails, that would be very helpful in diagnosing and fixing the issue. Thanks!

**Comment by DogancanYr** [comment]
I started Discord normally and it doesn't work. There is no error message either. I have to turn it off and on for it to work.

**Comment by webcrawls** [comment]
Could I get you to confirm you're on the latest version, `v1.2.1`? After giving Discord a full restart on my machine, the plugin seems to load fine as soon as BD is initialized, so I'm not totally sure what could be going on here.

If you _are_ on the latest version, it's possible that folder colouring code runs before Discord is able to add the folder elements. If you're up for it, I could send a test version of the plugin, adding a delay to the initial folder colouring stuff - a hacky solution, but the easiest to confirm that your issue is caused by slower folder loading.

**Comment by DogancanYr** [comment]
Yes I am using the latest version. I think it conflicts with other plugins and the best solution would be to add a delay.

**Comment by webcrawls** [comment]
[Here's a build of PrettyFolders that adds 5 seconds worth of delay to the colouring code](https://gist.github.com/webcrawls/17063cf1d2033b4e54927b6070d9b617). If you're unfamiliar with installing custom plugins:
- go to Discord's settings
- navigate to BetterDiscord > Plugins
- click the folder icon ('Open Plugins Folder')
- copy the testing build in, overwriting the existing `pretty-folders.plugin.js`
- restart your Discord

If that works, let me know and I can work on devising a better solution. I'd very much like to know what other plugins you have installed, as if there's a specific plugin causing issues, I can work out what's causing the conflict and try to fix it with a better method. Thanks!

**Comment by DogancanYr** [comment]
Yes, it worked this time, but I had to set the time to 12000, I think one plugin refreshes the left side somehow. (there are 50 plugins)

**Comment by webcrawls** [comment]
Glad to hear you got it working. I'm hoping to refactor the code soon, which should (hopefully) make it more resistant to conflicts - keep your eyes out for an update in the future. If you experience any other problems, or get an idea on what plugin could be causing the conflict, feel free to open another issue!

**Comment by DogancanYr** [comment]
Can you share it again?
